data/test/integrationTest/dataset_check.py

```diff
@@ -0,0 +1,72 @@
+import h5py
+import numpy as np
+import argparse
+
+from nose.tools import assert_equals, assert_true, assert_false,\
+    assert_equal, assert_not_equal
+
+
+class DatasetChecker:
+
+    def __init__(self, args):
+
+        # open file
+        # read datasets into a variable of some sort
+        with h5py.File(args.filename, "r") as data_file:
+            print("Keys: {}".format(data_file.keys()))
+
+            self.data = np.array(data_file[args.dataset_name])
+            self.raw = np.array(data_file["raw_frames"])
+            # print(self.data)
+
+    def check_averages(self):
+
```

Comment:
It all looks good but I'm just wondering whether the function should be named something else if you have any plans on implementing something similar for the Discrimination algorithm?